src/NimBLECharacteristic.h

```diff
@@ -83,9 +88,9 @@ class NimBLECharacteristic {
     void              indicate();
     void              indicate(const uint8_t* value, size_t length);
     void              indicate(const std::vector<uint8_t>& value);
-    void              notify(bool is_notification = true);
-    void              notify(const uint8_t* value, size_t length, bool is_notification = true);
-    void              notify(const std::vector<uint8_t>& value, bool is_notification = true);
+    void              notify(bool is_notification = true, int16_t conn_handle = (-1));
+    void              notify(const uint8_t* value, size_t length, bool is_notification = true, int16_t conn_handle = (-1));
+    void              notify(const std::vector<uint8_t>& value, bool is_notification = true, int16_t conn_handle = (-1));
```

Comment:
Instead of `int16_t` it would be better if we used `uint16_t` to match the types. The default value should be the max defined + 1, which is `BLE_HCI_LE_CONN_HANDLE_MAX`.

Reply:
corrected as per comment